### DISCUSSION TITLE
Copilot Extension Activation Failed

### DISCUSSION THREAD

**Discussion by patilyogeshp** [community/community#12277] [Copilot Conversations] [answered]
I am using VSCode. I am trying to active copilot in VSCode as I have received invite for Technical Preview. But it is failing with error _**GitHub Copilot could not connect to server. Extension activation failed: "getaddrinfo EAI_AGAIN api.github.com"**_

**Comment by davecheney** [comment]
Is your internet connection being filtered by your employer, internet service provider, or government?

**Comment by patilyogeshp** [comment]
I am using proxy to connect to internet.

**Reply by davecheney** [reply]
Sadly copilot does not support proxies at the moment.

**Reply by sxwebster** [reply]
It definitely needs to.

**Comment by patilyogeshp** [comment]
Exactly

**Comment by davecheney** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/13743
